src/domain/registration/form/RegistrationForm.tsx

```diff
@@ -1,5 +1,205 @@
+                    )}
+                  />
+                </div>
+              </div>
+
+              <div className={styles.guardianInfo}>
+                <Field
+                  type="text"
+                  name="guardian.email"
+                  label={formatMessage(
+                    'registration.form.guardian.email.input.label'
+                  )}
+                  onChange={handleChange}
+                  value={values.guardian.email}
+                  component={InputField}
+                  placeholder={formatMessage(
+                    'registration.form.guardian.email.input.placeholder'
+                  )}
+                />
+
+                <Field
+                  type="text"
+                  name="guardian.phoneNumber"
+                  label={formatMessage(
+                    'registration.form.guardian.phoneNumber.input.label'
+                  )}
+                  onChange={handleChange}
+                  value={values.guardian.phoneNumber}
+                  component={InputField}
+                  placeholder={formatMessage(
+                    'registration.form.guardian.phoneNumber.input.placeholder'
+                  )}
+                />
+                <div className={styles.guardianName}>
+                  <Field
+                    type="text"
+                    name="guardian.firstName"
+                    label={formatMessage(
+                      'registration.form.guardian.firstName.input.label'
+                    )}
+                    onChange={handleChange}
+                    value={values.guardian.firstName}
+                    component={InputField}
+                    placeholder={formatMessage(
+                      'registration.form.guardian.firstName.input.placeholder'
+                    )}
+                  />
+                  <Field
+                    type="text"
+                    name="guardian.lastName"
+                    label={formatMessage(
+                      'registration.form.guardian.lastName.input.label'
+                    )}
+                    onChange={handleChange}
+                    value={values.guardian.lastName}
+                    component={InputField}
+                    placeholder={formatMessage(
+                      'registration.form.guardian.lastName.input.placeholder'
+                    )}
+                  />
+                </div>
+                <Field
+                  name="guardian.relationship"
+                  label={formatMessage(
+                    'registration.form.guardian.relationship.input.label'
+                  )}
+                  onChange={handleChange}
+                  value={values.guardian.relationship}
+                  component={SelectField}
+                  id="registration.form.guardian.relationship.select"
+                  options={[
+                    { label: 'Parents', value: 'parents' },
```

Comment:
Any reason why label is not translated?

Reply:
its just a mock, will remove later when we go specific, but i can make a translation to be more legit code

Reply:
Ok, then it's probably unnecessary to translate it now